src/components/probeDetails.jsx

```diff
@@ -351,7 +353,7 @@ class ProbeDetails extends Component {
               {getExtraProbeDetails(probe, probeInfo).map(detail => (
                 <tr key={detail.label}>
                   <td className="fit pr-2">{detail.label}:</td>
-                  <td className="grow">{detail.content}</td>
+                  <td className="grow">{detail.content.toString()}</td>
```

Comment:
This fixes the bug of `Keyed` boolean field not showing. [The variables of `detail.content`](https://github.com/mozilla/probe-dictionary/blob/main/src/components/probeDetails.jsx#L201-L213) are all string-like values so adding an extra `toString()` step is safe.